packages/nimble-components/src/card/styles.ts

```diff
@@ -1,35 +1,34 @@
 import { css } from '@microsoft/fast-element';
 import { display } from '@microsoft/fast-foundation';
 import {
+    bodyEmphasizedFontWeight,
     bodyFont,
     bodyFontColor,
-    borderColor,
-    borderWidth,
-    sectionBackgroundColor,
-    standardPadding,
-    titleFont,
-    titleFontColor
+    cardBorderColor,
+    subtitleFontColor,
+    subtitlePlus1Font
 } from '../theme-provider/design-tokens';
 
 export const styles = css`
     ${display('flex')}
 
     :host {
         flex-direction: column;
-        gap: ${standardPadding};
-        padding: ${standardPadding};
-        border: ${borderWidth} solid ${borderColor};
+        gap: 24px;
```

Comment:
I didn't see an existing design token for 24px, either for this gap between controls or for the padding inside the border. Let me know if I should create one.

Reply:
Sounded like the expected direction from the notes captured here: https://github.com/ni/nimble/issues/1518#issuecomment-1716386610

We hadn't really figured out naming yet. Seems like large is pretty straightforward on the scale for 24px. How we handle beyond that isn't clear 
- extra-large >> extra-extra-large 
- x-large >> xx-large 
-  x-large >> 2x-large (might not be good for identifiers)
-  x-large >> x2-large
- large >> large-plus-1 >> large-plus-2 / small >> small-minus-1 >> small-minus-2

I dunno. That's a problem for future us. Yea 24px as large sounds good.

Reply:
I created a `largePadding` token and added it to the size ramp storybook page.

![image](https://github.com/ni/nimble/assets/35350751/9a6add73-9129-446d-a7b9-ee5d652f71fc)